src/osdc/Objecter.cc

```diff
@@ -1258,7 +1258,7 @@ void Objecter::handle_osd_map(MOSDMap *m)
       } else {
 	ldout(cct, 3) << "handle_osd_map hmm, i want a full map, requesting"
 		      << dendl;
-	monc->sub_want("osdmap", 0, CEPH_SUBSCRIBE_ONETIME);
+	monc->sub_want("osdmap", 0, CEPH_SUBSCRIBE_ONETIME|CEPH_SUBSCRIBE_LATEST_OSDMAP);
```

Comment:
This is requesting epoch 0.  How `CEPH_SUBSCRIBE_LATEST_OSDMAP` affects this?

Reply:
I think no. I want to be clear about my purpose.

Reply:
updated